charts/kubeclarity/values.yaml

```diff
@@ -282,13 +281,62 @@ kubeclarity-sbom-db:
 ## KubeClarity Postgres Values
 
 kubeclarity-postgresql:
-  enabled: true
```

Comment:
This structure is nice, it means that we can do some common things, I gave me the idea that we could do something like:

```
kubeclarity-postgresql:
   createPasswordSecret: true
   createPasswordSecretName: kubeclarity-postgresql-secret
   # Default is postgres-password
   # createPasswordSecretKey: 

   internal:
      ...
      auth:
        # If createPasswordSecret true, then should match createPasswordSecretName, 
        # otherwise set it to your own secret
        existingSecret: kubeclarity-postgresql-secret
   external:
      ...
      auth:
        # If createPasswordSecret true, then should match createPasswordSecretName
        # otherwise set it to your own secret
        existingSecret: kubeclarity-postgresql-secret
```

if kubeclarity-postgresql.createPasswordSecret == true, then create the secret with the password from global. databasePassword. That way we can control creating the secret separately from using the secret in the internal/external, the benefit here is we get the option to use an existing secret for internal postgres too if someone doesn't want to put their password in helm.

Reply:
Thanks for your input.
PR updated with your idea.